client/src/routes/automation/BlocklyComponent.svelte

```diff
@@ -61,41 +61,29 @@
     });
 </script>
 
-<div>
+<div class="pr-4">
     {#await entityPromise}
         <p>{tr('blockly.loading')}</p>
-        <div id="blocklyDivHide" />
+        <div
+            id="blocklyDivHideAwait"
```

Comment:
Where is this id used ?

Reply:
not used, just to make the code more readable